src/v/raft/consensus.cc

```diff
@@ -40,6 +41,15 @@
 
 namespace raft {
 
+static std::vector<model::record_batch_type>
+offset_translator_batch_types(const model::ntp& ntp) {
+    if (ntp.ns == model::kafka_namespace) {
+        return {model::record_batch_type::raft_configuration};
```

Comment:
This is probably not the right place to keep this list - I'd prefer it to be in higher layer. OTOH when I tried to move it somewhere higher the code looked really ugly. I'm open to suggestions on what the right place must be though.